pyproject.toml

```diff
@@ -28,7 +28,7 @@ classifiers=[
 dependencies = [
 "numpy>=1.26.4,<2.0",
 "accelerate>=0.20.3,<0.40",
-"transformers>=4.34.1,<=4.40.2,!=4.38.2",
+"transformers>=4.41.0,<5.0,!=4.38.2",
```

Comment:
@anhuong @Abhishek-TAMU 

transformers 4.41 has a severe performance degbradation, hence we want to keep it > 4.41 and not >=

also `!=4.38.2",` is not needed now as it is lesser than 4.41 and does not satisfy condition

Reply:
This new [PR](https://github.com/foundation-model-stack/fms-hf-tuning/pull/250/) reflects this change.